components/molecule/rating/src/index.scss

```diff
@@ -0,0 +1,84 @@
+@import '~@schibstedspain/sui-theme/lib/index';
+@import '~react-rating-svg/lib/scss/react-rating-svg';
+
+$fz-molecule-rating-small: $fz-s !default;
+$fz-molecule-rating-medium: $fz-m !default;
+$fz-molecule-rating-large: $fz-l !default;
+
+$c-molecule-rating: #5f5f5f !default;
+$c-molecule-rating-label-hover: #5398c1 !default;
+$bg-molecule-rating-hover: #ecf4f8 !default;
+
+$m-molecule-rating-label: 0 0 0 $m-m !default;
+
+$base-class: '.sui-MoleculeRating';
+$class-star: '#{$base-class}-Star';
+$class-label: '#{$base-class}-label';
+
+#{$base-class} {
+  align-items: center;
+  color: $c-molecule-rating;
+  display: flex;
+  
+  &-containerStars {
+    display: flex;
+  }
+
+  #{$class-label} {
+    display: flex;
+    margin: $m-molecule-rating-label;
+    padding: $p-m;
+
+    &:hover {
+      background: $bg-molecule-rating-hover;
+      color: $c-molecule-rating-label-hover;
+      cursor: default;
+    }
+  }
+
+  &--small {
+    font-size: $fz-molecule-rating-small;
+
+    #{$class-star} {
+      height: 24px;
```

Comment:
I think this should be one of our icon sizes

Reply:
fixed!